app/controllers/CustomsRegulationTypeController.scala

```diff
@@ -18,10 +18,12 @@ package controllers
 
 import controllers.actions._
 import forms.CustomsRegulationTypeFormProvider
+import models.CustomsRegulationType.UKCustomsCodeRegulation
+
 import javax.inject.Inject
-import models.{Mode, NumberOfEntriesType, UserAnswers}
+import models.{CustomsRegulationType, Mode, NumberOfEntriesType, UserAnswers}
 import navigation.Navigator
-import pages.{CustomsRegulationTypePage, NumberOfEntriesTypePage}
+import pages.{ArticleTypePage, CustomsRegulationTypePage, NumberOfEntriesTypePage, UkRegulationTypePage}
```

Comment:
Why have these been added? I see no new reference to them.

Reply:
@lmugnai this is fixed